### PR TITLE
Change opening hours of Leicester Citizens Advice

### PR DESCRIPTION
**Pull request by jiannis**

### CODE REVIEW

**Comment by jiannis** [comment]
@andrewgarner Just deployed a small change to the opening hours of Leicester Citizens Advice. We need to clear the cache for [this page](https://www.pensionwise.gov.uk/locations/fdcde285-37fd-4858-b00a-4ee31b4488b1) :camel:

File: public/js/cita.json [hunk 1]
```diff
@@ -1710,7 +1710,7 @@
     "title": "Leicester",
     "address": "3rd Floor\n60 Charles Street\nLEICESTER\nLE1 1FB",
     "phone": "0116 326 6326",
-    "hours": "Monday to Friday, 9am to 5pm",
+    "hours": "Monday to Friday, 9am to 15:30pm",
```
Comment (andrewgarner) [inline public/js/cita.json:1713]: We use a 12 clock. This should read "3:30pm".

Comment (jiannis) [inline public/js/cita.json:1713]: Oops! That was a 36 hour clock :palm_tree:

Comment (jiannis) [inline public/js/cita.json:1713]: Fixed in 07f96926 :sunny:

Comment (andrewgarner) [inline public/js/cita.json:1713]: :+1:

Comment (jiannis) [inline public/js/cita.json:1713]: Clear cache for [this page](https://www.pensionwise.gov.uk/locations/fdcde285-37fd-4858-b00a-4ee31b4488b1) :bikini: ?

Comment (jiannis) [inline public/js/cita.json:1713]: @andrewgarner thank you :ok_hand:

**Pull request merged**